index.d.ts

```diff
@@ -29,25 +27,27 @@ declare module 'data-store' {
     constructor (name?: string, options?: Options, defaults?: any)
     constructor (options?: Options, defaults?: any)
 
+    readonly data: DataObject
+
     /**
      * Assign value to key and save to the file system.
      * Can be a key-value pair, array of objects, or an object.
      */
 
-    set: (key: string, val: any) => Store
+    set: (key: string, val: any) => this
```

Comment:
Is using `this` a short cut for specifying the type?

Reply:
https://www.typescriptlang.org/docs/handbook/advanced-types.html#polymorphic-this-types